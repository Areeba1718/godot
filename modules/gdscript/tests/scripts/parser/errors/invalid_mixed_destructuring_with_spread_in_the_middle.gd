func test():
    var [a, ..rest, b] = [1, 2, 3, 4]
