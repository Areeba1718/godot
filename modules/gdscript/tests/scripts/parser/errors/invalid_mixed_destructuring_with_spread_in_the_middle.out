GDTEST_PARSER_ERROR
Only the last element in a destructuring expression can have the spread qualifier ("..").
